lms/djangoapps/xblock_user_state/interface.py

```diff
@@ -1,255 +0,0 @@
-"""
-A baseclass for a generic client for accessing XBlock Scope.user_state field data.
-"""
-
-from abc import abstractmethod
-
-from contracts import contract, new_contract, ContractsMeta
-from opaque_keys.edx.keys import UsageKey
-from xblock.fields import Scope, ScopeBase
-
-new_contract('UsageKey', UsageKey)
-
-
-class XBlockUserStateClient(object):
```

Comment:
You'll need to change the places that import this class to import the new module name.